README.md

```diff
@@ -4,7 +4,7 @@
 
 # `staramr`
 
-`staramr` (*AMR) scans bacterial genome contigs against both the [ResFinder][resfinder-db] and [PointFinder][pointfinder-db] databases (used by the [ResFinder webservice][resfinder-web]) and compiles a summary report of detected antimicrobial resistance genes.
+`staramr` (*AMR) scans bacterial genome contigs against the [ResFinder][resfinder-db], [PointFinder][pointfinder-db], and [PlasmidFinder][plasmidfinder-db] databases (used by the [ResFinder webservice][resfinder-web]) and compiles a summary report of detected antimicrobial resistance genes.
```

Comment:
Could you change:

* used by the ResFinder webservice -> used by ResFinder **and other webservices offered by the Center for Genomic Epidemiology)**

This is because PlasmidFinder is not part of ResFinder (but is part of the larger suite of tools offered by the website).

Reply:
Changed this in commit 2202e45

Reply:
Thanks :smile: